fenix/app/src/main/java/org/mozilla/fenix/browser/BaseBrowserFragment.kt

```diff
@@ -480,13 +481,20 @@ abstract class BaseBrowserFragment :
                 )
             }
 
-            BottomToolbarContainerView(
+            bottomToolbarContainerView = BottomToolbarContainerView(
                 context = context,
                 container = binding.browserLayout,
                 androidToolbarView = if (isToolbarAtBottom) browserToolbar else null,
                 menuButton = menuButton,
                 browsingModeManager = activity.browsingModeManager,
             )
+
+            requireComponents.appStore.flowScoped { flow ->
```

Comment:
We will want to also do this via a `LifecycleAwareFeature` pattern (eg, `FindInPageIntegration`) so that we aren't calling `appStore.flowScoped` directly here.